src/lib/dnssd/Types.h

```diff
@@ -205,7 +205,7 @@ inline constexpr size_t kMaxRotatingIdLen         = 50;
 inline constexpr size_t kMaxPairingInstructionLen = 128;
 
 /// Data that is specific to commisionable/commissioning node discovery
-struct CommissionNodeData
+struct DnssdNodeData
```

Comment:
The struct name is quite vague as `OperationalNodeData` is technically also related to DNS-SD but I can't come up with a better alternative (though the structure seems to contain most PTR+SRV data so maybe something like `NodeServiceData` would make sense...) so treat it as a non-blocking comment.

Reply:
There will be a follow-up PR where this structure will also be used for Operational Browse. That is the reason this was named as DnssdNodeData as it will be common for all node type browse delegates.

Reply:
Combined PR for your reference : #32750